posix/subsystem/src/fs.cpp

```
@@ -25,6 +29,10 @@ async::result<frg::expected<Error, FileStats>> FsNode::getStats() {
 	throw std::runtime_error("getStats() is not implemented for this FsNode");
 }
 
+async::result<frg::expected<Error, FsFileStats>> FsNode::getFsstats() {
+	throw std::runtime_error("getFsstats() is not implemented for this FsNode");
```

This should return an error, not blow up posix.